plugins/contact-resources/src/components/EmployeeAccountPresenter.svelte

```diff
@@ -48,7 +49,7 @@
   >
     {#if employee}
       <Avatar size={'x-small'} avatar={employee.avatar} />
-      <div class="overflow-label user">{getName(employee)}</div>
+      <div class="overflow-label user" class:user-enabled={!disabled}>{getName(employee)}</div>
```

Comment:
Could you please use DocNavLink (I forget add it here), it already have disabled param

Reply:
Done